dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -69,6 +69,36 @@ def test_profiled_precision(self):
         float_profiler.update(df_mix)
         self.assertEqual(1, float_profiler.precision)
 
+        # edge cases #
+        # integer with 0s on right and left side
+        df_ints = pd.Series(['0013245678', '123456700', '0012345600'])
+        float_profiler = FloatColumn("Name")
+        float_profiler.update(df_ints)
+        self.assertEqual(6, float_profiler.precision)
+
+        # scientific
+        df_scientific = pd.Series(['1.23e-3', '2.2344', '1.244e4'])
+        float_profiler = FloatColumn("Name")
+        float_profiler.update(df_scientific)
+        self.assertEqual(3, float_profiler.precision)
+
+        # plus
+        df_plus = pd.Series(['+1.3e-3', '+2.244', '+1.3324e4'])
+        float_profiler = FloatColumn("Name")
+        float_profiler.update(df_plus)
+        self.assertEqual(2, float_profiler.precision)
+
+        # minus
+        df_minus = pd.Series(['-1.3234e-3', '-0.244', '-1.3324e4'])
+        float_profiler = FloatColumn("Name")
+        float_profiler.update(df_minus)
+        self.assertEqual(3, float_profiler.precision)
+
+        # spaces around values
+        df_spaces = pd.Series(['  -1.3234e-3  ', '  -0.244  '])
```

Comment:
What precision does it think '  -1.3234e-3  ' is?
Can we have a test for something like 323000.? 
I'm slightly worried that some of these tests actually fail, but since we are testing multiple floats at the same time, it still passes. Like maybe it is treating '  -1.3234e-3  ' as 12 sig figs (exaggeration i know), but the test passes because it properly recognized the common denominator of 3 with '  -0.244  '. Thoughts? I think we might want to separate many of these tests out

Reply:
fair, I will have to loop through since the function takes a series and provides the minimum.